vault/resource_azure_secret_backend.go

```diff
@@ -164,8 +157,19 @@ func azureSecretBackendRead(d *schema.ResourceData, meta interface{}) error {
 		d.Set("environment", "AzurePublicCloud")
 	}
 
-	d.Set("path", path)
-	d.Set("description", mount.Description)
+	if v, ok := resp.Data["use_microsoft_graph_api"]; ok {
```

Comment:
nit: I know we're wanting to scope things as relevant as possible to only changes we make, but I'm wondering if this is a good opportunity to also add better error handling for lines 145-157 (`d.Set` for the other fields).

Reply:
Good call!

Reply:
Addressed in https://github.com/hashicorp/terraform-provider-vault/pull/1335/commits/aabd53948bc417bd0a599b7fbe3754272c39b9a4